### PR TITLE
[Feature] Custom remedies

### PR DESCRIPTION
**Pull request by jorGonzalez291292**
## Descripción
New displayInfo node is added to draw the congrats header when there are remedies

## Screenshots
Screenshot_20220120-125609_MP Android SDK Samples

### CODE REVIEW

File: px-checkout/src/main/java/com/mercadopago/android/px/internal/features/payment_result/mappers/PaymentResultHeaderModelMapper.java [hunk 1]
```diff
@@ -55,16 +58,63 @@
...
     }
 
+    private String resolveTitle(final RemediesModel remediesModel) {
+        String title = getInstructionsTitle();
+
+        if (remediesModel.hasRemedies() && remediesModel.getHeader() != null) {
+            title = remediesModel.getHeader().getTitle();
+        }
+
+        return title;
```
Comment (daniivillegas) [inline px-checkout/src/main/java/com/mercadopago/android/px/internal/features/payment_result/mappers/PaymentResultHeaderModelMapper.java:78]: no podriamos hacer el if ternario aca?

File: px-checkout/src/main/java/com/mercadopago/android/px/internal/features/payment_result/mappers/PaymentResultHeaderModelMapper.java [hunk 2]
```diff
@@ -55,16 +58,63 @@
...
+
+    @ColorRes
+    private int resolveBackground(final PaymentResultViewModel viewModel) {
+        int backgroundResId = viewModel.getBackgroundResId();
+
+        if (remediesModel.hasRemedies()) {
+            backgroundResId = RemediesModel.DECORATOR.resColor;
+        }
+
+        return backgroundResId;
```
Comment (daniivillegas) [inline px-checkout/src/main/java/com/mercadopago/android/px/internal/features/payment_result/mappers/PaymentResultHeaderModelMapper.java:94]: idem que arriba :P

File: px-checkout/src/main/java/com/mercadopago/android/px/internal/features/payment_result/mappers/PaymentResultHeaderModelMapper.java [hunk 3]
```diff
@@ -55,16 +58,63 @@
...
+    }
+
+    private String resolveBadgeUrl(final RemediesModel remediesModel) {
+        String iconUrl = "";
+
+        if (remediesModel.hasRemedies() && remediesModel.getHeader() != null) {
+            iconUrl = remediesModel.getHeader().getBadgeUrl();
+        }
+
+        return iconUrl;
```
Comment (daniivillegas) [inline px-checkout/src/main/java/com/mercadopago/android/px/internal/features/payment_result/mappers/PaymentResultHeaderModelMapper.java:115]: idem que arriba :P

File: px-checkout/src/main/java/com/mercadopago/android/px/internal/view/PaymentResultHeader.java [hunk 4]
```diff
@@ -68,13 +68,26 @@ private void renderIcon(@NonNull final ImageView icon, @NonNull final Model mode
             .into(icon);
     }
 
+    private void renderBadge(@NonNull final ImageView badge, @NonNull final Model model) {
+        final int size = ScaleUtil.getPxFromDp(90, getContext());
```
Comment (RoBarriosG) [inline px-checkout/src/main/java/com/mercadopago/android/px/internal/view/PaymentResultHeader.java:72]: Minor comment, 90 should be a class constant

**Review by daniivillegas — APPROVED** [review]

File: px-checkout/src/main/java/com/mercadopago/android/px/internal/view/PaymentResultHeader.java (hunk 4 above)
Comment (jorGonzalez291292) [reply px-checkout/src/main/java/com/mercadopago/android/px/internal/view/PaymentResultHeader.java]: updateado

**Review by RoBarriosG — APPROVED** [review]

**Pull request merged**